### PR TITLE
Feature cf

### PR DESCRIPTION
**Pull request by cmfung**
@boyiq @avaboonzaayer fixed the cards and columns autosizing incorrectly, added hover effects to cards, removed strict column to allow for autocalculating, implemented card sizes with 'vw'.

### CODE REVIEW

File: client/dist/style.css [hunk 1]
```diff
@@ -229,57 +229,6 @@ body, button {
 
 /********************************************/
 
-.star {
```
Comment (boyiq) [inline client/dist/style.css]: Were these star styles accidentally removed?

**Review by boyiq — COMMENTED** [review]
Everything else looks good except the comment in the css file

File: client/dist/style.css (hunk 1 above)
Comment (cmfung) [reply client/dist/style.css]: @boyiq  I didn't make any changes after merging, but that def looks incorrect so something might have gone wrong with my merge. Let's look together when we hop on in a bit

**Review by boyiq — APPROVED** [review]
Looks good now!

**Pull request merged**